src/components/filmDetailsPage/SimilarList.vue

```diff
@@ -1,70 +1,29 @@
 <template>
   <div class="films-list">
-    <div class="film-card" v-for="film in films" v-bind:key="film.title">
-      <router-link to="/film-details"
-        ><FilmCard
-          :imageSrc="film.image"
+    <div class="film-card" v-for="film in movies" v-bind:key="film.id">
+      <router-link :to="{ name: 'Film details', params: { id: film.id } }">
```

Comment:
Consider creating "routeCreators" file under "src/router" and keep such logic in there.
Additionally, consider creating "routes" file under the same folder to keep names of the routes as constants and reusing them across the app.
E.g. `const filmDetailsRoute = (id) => ({ name: ROUTES.FILM_DETAILS, params: { id } });`

Reply:
Fixed